src/boot/chid.c

```diff
@@ -88,7 +91,7 @@ static EFI_STATUS populate_board_chids(EFI_GUID ret_chids[static CHID_TYPES_MAX]
         return EFI_SUCCESS;
 }
 
-EFI_STATUS chid_match(const void *hwid_buffer, size_t hwid_length, const Device **ret_device) {
+EFI_STATUS chid_match(const void *hwid_buffer, size_t hwid_length, uint32_t match_type, const Device **ret_device) {
```

Comment:
Move this change to 458945ff7629b7c0028e843221e6caa1f98e4846
It's not directly related to the new device type and isn't used anywhere here and really belongs there. That commit also needs to be first.

Reply:
Again, I would argue that all these changes could have been part of a separate PR since you are asking them to move to a different commit. I do not know what morale boost you are expecting from a person who is trying to merge something for more than a month. When a PR gets merged, progress is made. Incremental fixes can be made on top of that change. I have contributed to several communities where this thought prevails. When you put someone in an endless cycle of changes, they lose their morale, they are frustrated and they are certainly not encouraged to make more contributions. @poettering if we meet in FOSDEM, we can talk about this more in person.